app/Console/Commands/MigrateYearPeriodYearToVacationLimits.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Toby\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\DB;
+
+class MigrateYearPeriodYearToVacationLimits extends Command
+{
+    protected $signature = "toby:migrate-year-period-year-to-vacation-limits";
+    protected $description = "Migrate year period to vacation limits";
+
+    public function handle(): void
+    {
+        DB::statement("
```

Comment:
What would happen if you'd run this twice? Maybe it would be good to make some if statement here to avoid that.